benchmark/data_service.py

```diff
@@ -292,10 +292,30 @@ def convert_to_zarr(input_vcf_path, output_zarr_path, conversion_config):
         # Ensure var is string, not pathlib.Path
         output_zarr_path = str(output_zarr_path)
 
-        # Scan VCF file to find max number of alleles in any variant
-        callset = allel.read_vcf(input_vcf_path, fields=['numalt'], log=sys.stdout)
-        numalt = callset['variants/numalt']
-        alt_number = np.max(numalt)
+        # Get alt number
+        if conversion_config.alt_number is None:
+            print("[VCF-Zarr] Determining alt number from VCF meta-information.")
```

Comment:
Very minor point, but "meta-information" would usually be understood to mean the file headers, whereas here we're performing a full scan of the VCF file to find the variant with the largest number of ALT alleles. Maybe something like "Determining maximum number of ALT alleles by scanning all variants the VCF file."?